google/cloud/bigtable/row_key.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_ROW_KEY_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_ROW_KEY_H_
```

Comment:
I've no real preference here, but do you think it's worth defining this in a separate row_key.h file, vs moving this alias declaration into row.h, and having callers include that if they need this the Row's key type? I'm really not pushing one way or the other, just wondering.

Reply:
I have a slight preference for keeping as-is, but won't matter that much either way .  `RowKeyType` is used by `mutations.h`, `row.h`, `cell.h`, `row_key_sample.h`, it felt more reasonable to refactor to a single place instead of having one of the larger includes be used by all the others.

Reply:
keeping as-is SGTM. Thanks.